druid/src/window.rs

```diff
@@ -182,6 +182,50 @@ impl<T: Data> Window<T> {
                 false,
             );
         }
+
+        // Update the disabled state if necessary
+        // Always do this before sending focus change, since this event updates the focus chain.
+        if self.root.state().tree_disabled_changed() {
+            let event = LifeCycle::Internal(InternalLifeCycle::RouteDisabledChanged);
+            self.lifecycle(queue, &event, data, env, false);
```

Comment:
This could be what happened to me, when i was talking about the stackoverflow on Zulip.
Hmm :/ Calling `children_changed()` in `DisabledChanged` and `set_disabled()` in `RouteWidgetAdded`, will definitively result in an infinite loop. Maybe we should just warn about this for now and fix it in #1603?